### DISCUSSION TITLE
有一些小需求

### DISCUSSION THREAD

**Discussion by nikissXI** [linweiyuan/go-chatgpt-api#55] [Q&A] [answered]
增加两个接口，有时候需要清理一些会话，都是一个名字不好管理
1、给会话重命名
2、删除会话

大佬有打算嘛

**Comment by linweiyuan** [comment]
这两功能从项目出生开始就有了，和官方接口参数一样，另外还有删除单个会话、删除所有会话、给消息反馈（点赞点踩）都是可用的，参考：https://linweiyuan.github.io/2023/02/25/ChatGPT-Swing.html

**Reply by nikissXI** [reply]
woc，6啊

**Reply by nikissXI** [reply]
是不是没做接口文档，我看不懂JAVA代码，qwq，能给重命名和删除的请求示例吗

**Reply by linweiyuan** [reply, marked as answer]
删除会话 `PATCH` `/conversation/{conversationId}`

```json
{
  "is_visible": false
}
```

重命名相同链接

```json
{
  "title": "xxxxx"
}
```


应该没错

**Reply by nikissXI** [reply]
方法都是POST，我翻了你这个项目的代码ChatGPT-Swing

**Reply by linweiyuan** [reply]
Java 不直接支持 `PATCH`

你可以看 go-chatgpt：https://github.com/linweiyuan/go-chatgpt/blob/main/api/api.go#L190

**Reply by nikissXI** [reply]
噢噢。。。我用POST也能正常请求
